pkg/asset/cluster/baremetal/baremetal.go

```diff
@@ -10,7 +10,8 @@ import (
 // Metadata converts an install configuration to bare metal metadata.
 func Metadata(config *types.InstallConfig) *baremetal.Metadata {
 	return &baremetal.Metadata{
-		LibvirtURI: config.Platform.BareMetal.LibvirtURI,
-		IronicURI:  config.Platform.BareMetal.IronicURI,
+		LibvirtURI:              config.Platform.BareMetal.LibvirtURI,
+		BootstrapProvisioningIP: config.Platform.BareMetal.BootstrapProvisioningIP,
```

Comment:
why is the `BootstrapProvisioningIP` and `ClusterProvisioningIP` required in the metadata?

Reply:
I can't think of a reason to keep it, will remove it. AFAIK this data is really only preserved for cluster destruction, but the name isn't really clear.  I've seen several people (including me in beginning) adding fields to the install-config repeatedly confused by this and try to add all the data here. Not sure if that's common for others but it could use a different name, or explict comments everywhere saying what it's for.

Reply:
Yeah it's probably not needed, it got changed here because IronicURI was already there, and I modified that existing name to BootstrapProvisioningIP via sed and some manual formatting fixes.